resources/core/charts/gateway/templates/application-connector-certs.yaml

```diff
@@ -1,8 +1,10 @@
 ---
 apiVersion: v1
 data:
```

Comment:
Note to self : we need to check if we can reuse kyma secret or issue new cert from Gardener directly on gateway.$DOMAIN